### PR TITLE
(VDB-1396) Add auction file transformer

### PR DESCRIPTION
**Pull request by rmulhol**

### CODE REVIEW

File: environments/docker.toml [hunk 1]
```diff
@@ -239,6 +240,14 @@
         repository = "github.com/makerdao/vdb-mcd-transformers"
         migrations = "db/migrations"
         rank = "0"
+    [exporter.auction_file]
+        path = "transformers/events/auction_file/initializer"
+        type = "eth_event"
+        repository = "github.com/makerdao/vdb-mcd-transformers"
+        migrations = "db/migrations"
+        contracts = ["MCD_FLAP", "MCD_FLIP_BAT_A", "MCD_FLIP_ETH_A", "MCD_FLIP_SAI", "MCD_FLIP_TUSD_A",
```
Comment (yaoandrew) [inline environments/docker.toml]: 🎗️ just a note to remember to add new collateral types in the rebase.

File: transformers/events/auction_file/transformer.go [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+	"github.com/makerdao/vulcanizedb/pkg/core"
+	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres"
+)
+
+type Transformer struct{}
+
+func (t Transformer) ToModels(_ string, logs []core.EventLog, db *postgres.DB) ([]event.InsertionModel, error) {
+	var models []event.InsertionModel
+	for _, log := range logs {
+		err := shared.VerifyLog(log.Log, shared.ThreeTopicsRequired, shared.LogDataNotRequired)
```
Comment (yaoandrew) [inline transformers/events/auction_file/transformer.go]: I forget how we determine how many topics are required, but should this be `FourTopicsRequired`? `Signature` [0], `msg.sender` [1], `arg1` [2], and `arg2` [3]?

File: transformers/integration_tests/auction_file_test.go [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+
+	Context("Flop file events", func() {
+		msgSender := "0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB"
+		what := "pad"
+		data := "1200000000000000000"
+		auctionFileIntegrationTest(int64(9017707), test_data.FlopAddress(), msgSender, what, data)
+	})
+})
+
+func auctionFileIntegrationTest(blockNumber int64, contractAddressHex, msgSenderAddressHex, what, data string) {
```
Comment (yaoandrew) [inline transformers/integration_tests/auction_file_test.go:76]: 💯

**Review by yaoandrew — APPROVED** [review]
Nothing merge blocking, just had a few questions.

File: environments/docker.toml (hunk 1 above)
Comment (rmulhol) [reply environments/docker.toml]: good call!

File: transformers/events/auction_file/transformer.go (hunk 2 above)
Comment (rmulhol) [reply transformers/events/auction_file/transformer.go]: yes - nice catch!

**Pull request merged**